ShopLens/ShopLens.sln

```diff
@@ -1,6 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ImageRecognition", "ImageRecognition\ImageRecognition.csproj", "{3B726F99-F999-40F3-AAC4-4CA6AC69B2FF}"
+# Visual Studio 15
+VisualStudioVersion = 15.0.28107.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ImageRecognition", "ImageRecognition\ImageRecognition.csproj", "{3B726F99-F999-40F3-AAC4-4CA6AC69B2FF}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VoiceRecognition", "VoiceRecognition\VoiceRecognition.csproj", "{A1CF8A54-1E07-4F88-BD82-9D8CB970C864}"
```

Comment:
Extra project "VoiceRecognition"? Did you forget to add the project to git or just a leftover reference?

The branch doesn't contain such project code.

Reply:
I just added the project to the solution and deliberately didn't commit it. I'm planning to commit it to a new branch and then merge it with "WindowsFormsApplication".